2024-01-26 17:08:45,812 INFO    StreamThr :3042422 [internal.py:wandb_internal():86] W&B internal server running at pid: 3042422, started at: 2024-01-26 17:08:45.811495
2024-01-26 17:08:45,831 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status
2024-01-26 17:08:45,845 INFO    WriterThread:3042422 [datastore.py:open_for_write():87] open: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/run-8pvkre8x.wandb
2024-01-26 17:08:45,846 DEBUG   SenderThread:3042422 [sender.py:send():382] send: header
2024-01-26 17:08:45,899 DEBUG   SenderThread:3042422 [sender.py:send():382] send: run
2024-01-26 17:08:46,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: check_version
2024-01-26 17:08:46,861 INFO    SenderThread:3042422 [dir_watcher.py:__init__():211] watching files in: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files
2024-01-26 17:08:46,862 INFO    SenderThread:3042422 [sender.py:_start_run_threads():1136] run started: 8pvkre8x with start time 1706260125.838371
2024-01-26 17:08:46,862 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: check_version
2024-01-26 17:08:47,343 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: run_start
2024-01-26 17:08:47,355 DEBUG   HandlerThread:3042422 [system_info.py:__init__():27] System info init
2024-01-26 17:08:47,355 DEBUG   HandlerThread:3042422 [system_info.py:__init__():42] System info init done
2024-01-26 17:08:47,355 INFO    HandlerThread:3042422 [system_monitor.py:start():194] Starting system monitor
2024-01-26 17:08:47,385 INFO    SystemMonitor:3042422 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-01-26 17:08:47,386 INFO    HandlerThread:3042422 [system_monitor.py:probe():214] Collecting system info
2024-01-26 17:08:47,409 INFO    SystemMonitor:3042422 [interfaces.py:start():190] Started cpu monitoring
2024-01-26 17:08:47,443 INFO    SystemMonitor:3042422 [interfaces.py:start():190] Started disk monitoring
2024-01-26 17:08:47,471 INFO    SystemMonitor:3042422 [interfaces.py:start():190] Started gpu monitoring
2024-01-26 17:08:47,498 INFO    SystemMonitor:3042422 [interfaces.py:start():190] Started memory monitoring
2024-01-26 17:08:47,520 INFO    SystemMonitor:3042422 [interfaces.py:start():190] Started network monitoring
2024-01-26 17:08:47,542 DEBUG   HandlerThread:3042422 [system_info.py:probe():151] Probing system
2024-01-26 17:08:47,589 DEBUG   HandlerThread:3042422 [system_info.py:_probe_git():136] Probing git
2024-01-26 17:08:47,620 DEBUG   HandlerThread:3042422 [system_info.py:_probe_git():144] Probing git done
2024-01-26 17:08:47,621 DEBUG   HandlerThread:3042422 [system_info.py:probe():199] Probing system done
2024-01-26 17:08:47,621 DEBUG   HandlerThread:3042422 [system_monitor.py:probe():223] {'os': 'Linux-5.15.0-91-generic-x86_64-with-glibc2.17', 'python': '3.8.17', 'heartbeatAt': '2024-01-26T09:08:47.542568', 'startedAt': '2024-01-26T09:08:45.790164', 'docker': None, 'cuda': None, 'args': ('--local_rank=2', '--config_file', 'config/bert_pretraining_phase2_config.json', '--input_dir', '/home/yanxindong/mkor/bert/data/encoded/sequences_lowercase_max_seq_len_512_next_seq_task_true', '--output_dir', 'results/adam', '--weight_decay', '0.01', '--num_steps_per_checkpoint', '200'), 'state': 'running', 'program': 'run_pretraining.py', 'codePathLocal': 'run_pretraining.py', 'codePath': 'bert/run_pretraining.py', 'git': {'remote': 'https://github.com/kkkjj-d/mkor.git', 'commit': 'efe99e84fef5594780cfb1df6d6a1f61da4ebba0'}, 'email': '1262670315@qq.com', 'root': '/home/yanxindong/mkor', 'host': 'a6000-X640-G40', 'username': 'yanxindong', 'executable': '/home/yanxindong/anaconda/envs/eva/bin/python', 'cpu_count': 64, 'cpu_count_logical': 128, 'cpu_freq': {'current': 2053.125, 'min': 800.0, 'max': 3400.0}, 'cpu_freq_per_core': [{'current': 3300.002, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3100.0, 'min': 800.0, 'max': 3400.0}, {'current': 1300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 1700.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 3000.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 1800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 840.809, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 1225.787, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3092.752, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 1378.181, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 1400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 1700.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 1400.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 1000.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 1100.0, 'min': 800.0, 'max': 3400.0}, {'current': 2100.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 900.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3299.996, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 2700.0, 'min': 800.0, 'max': 3400.0}, {'current': 1400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3155.872, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.004, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3300.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 2200.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 1400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 3400.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}, {'current': 800.0, 'min': 800.0, 'max': 3400.0}], 'disk': {'/': {'total': 438.05088806152344, 'used': 28.042720794677734}}, 'gpu': 'NVIDIA RTX A6000', 'gpu_count': 8, 'gpu_devices': [{'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}, {'name': 'NVIDIA RTX A6000', 'memory_total': 51527024640}], 'memory': {'total': 503.51250076293945}}
2024-01-26 17:08:47,621 INFO    HandlerThread:3042422 [system_monitor.py:probe():224] Finished collecting system info
2024-01-26 17:08:47,621 INFO    HandlerThread:3042422 [system_monitor.py:probe():227] Publishing system info
2024-01-26 17:08:47,621 DEBUG   HandlerThread:3042422 [system_info.py:_save_conda():208] Saving list of conda packages installed into the current environment
2024-01-26 17:08:47,863 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/conda-environment.yaml
2024-01-26 17:08:57,864 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/conda-environment.yaml
2024-01-26 17:08:58,086 DEBUG   HandlerThread:3042422 [system_info.py:_save_conda():220] Saving conda packages done
2024-01-26 17:08:58,087 INFO    HandlerThread:3042422 [system_monitor.py:probe():229] Finished publishing system info
2024-01-26 17:08:58,093 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:08:58,098 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:08:58,099 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:08:58,099 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:08:58,099 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: python_packages
2024-01-26 17:08:58,099 DEBUG   SenderThread:3042422 [sender.py:send():382] send: files
2024-01-26 17:08:58,099 INFO    SenderThread:3042422 [sender.py:_save_file():1403] saving file wandb-metadata.json with policy now
2024-01-26 17:08:58,100 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: python_packages
2024-01-26 17:08:58,137 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:08:58,138 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:08:58,162 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:08:58,506 DEBUG   SenderThread:3042422 [sender.py:send():382] send: telemetry
2024-01-26 17:08:58,864 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/requirements.txt
2024-01-26 17:08:58,864 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/wandb-metadata.json
2024-01-26 17:08:58,976 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51d4031790>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:08:58,976 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:08:58,976 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:09:00,061 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47d3f40>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:09:00,061 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:09:00,061 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:09:01,507 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:02,147 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47e50d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:09:02,147 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:09:02,148 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:09:02,148 INFO    wandb-upload_0:3042422 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f51c47e50d0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 799, in urlopen
    retries = retries.increment(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47e50d0>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2758, in upload_file
    response = self._upload_file_session.put(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47e50d0>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2789, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2758, in upload_file
    response = self._upload_file_session.put(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-01-26 17:09:06,507 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:06,609 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47d32b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:09:06,609 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:09:06,609 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:09:11,508 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:13,118 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:09:13,118 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:09:13,162 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:09:15,835 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47e5400>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:09:15,835 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:09:15,835 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:09:17,431 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:17,870 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/config.yaml
2024-01-26 17:09:22,834 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:27,922 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:28,118 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:09:28,118 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:09:28,166 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:09:32,135 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47e5fa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:09:32,135 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:09:32,135 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:09:33,483 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:38,484 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:43,118 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:09:43,119 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:09:43,162 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:09:46,128 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:47,521 DEBUG   SystemMonitor:3042422 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-01-26 17:09:47,524 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:09:51,526 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:56,527 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:09:58,119 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:09:58,119 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:09:58,162 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:10:02,455 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:07,455 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:09,703 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c463d040>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:10:09,703 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:10:09,703 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:10:12,464 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:13,119 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:10:13,119 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:10:13,206 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:10:17,495 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:17,526 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:10:22,527 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:27,527 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:28,119 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:10:28,119 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:10:28,146 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:10:33,286 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:34,049 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/output.log
2024-01-26 17:10:36,050 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/output.log
2024-01-26 17:10:38,287 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:43,287 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:43,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:10:43,849 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:10:43,853 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:10:47,670 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:10:48,671 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:53,674 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:58,742 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:10:58,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:10:58,850 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:10:59,006 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:11:04,282 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:09,438 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:13,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:11:13,848 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:11:13,851 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:11:15,195 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:17,722 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:11:18,787 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c463dfa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:11:18,787 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:11:18,787 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:11:20,781 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:25,976 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:28,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:11:28,848 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:11:28,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:11:31,250 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:36,250 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:41,251 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:43,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:11:43,848 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:11:43,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:11:47,164 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:47,725 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:11:52,727 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:57,728 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:11:58,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:11:58,848 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:11:58,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:12:03,149 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:08,223 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:13,300 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:13,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:12:13,849 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:12:13,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:12:17,727 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:12:18,728 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:23,739 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:28,765 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:28,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:12:28,849 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:12:28,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:12:34,263 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:39,305 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:43,874 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:12:43,874 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:12:43,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:12:45,235 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:47,730 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:12:50,731 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:55,732 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:12:58,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:12:58,849 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:12:58,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:13:01,184 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:06,186 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:11,187 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:13,849 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:13:13,849 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:13:13,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:13:16,189 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:17,732 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:13:21,734 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:26,860 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:28,906 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:13:28,907 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:13:28,908 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:13:32,274 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:37,298 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:42,408 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:43,865 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:13:43,866 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:13:43,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:13:47,734 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:13:47,735 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:52,826 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:57,826 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:13:58,719 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47e5d90>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:13:58,720 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:13:58,720 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:13:58,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:13:58,850 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:13:58,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:14:03,213 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:08,213 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:13,214 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:13,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:14:13,850 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:14:13,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:14:17,736 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:14:18,738 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:23,738 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:28,761 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:28,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:14:28,850 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:14:28,894 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:14:34,294 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:39,454 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:43,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:14:43,850 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:14:43,894 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:14:45,191 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:47,756 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:14:50,942 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:55,963 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:14:58,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:14:58,850 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:14:58,894 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:15:01,180 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:15:06,233 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:15:11,234 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:15:13,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: stop_status
2024-01-26 17:15:13,850 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: stop_status
2024-01-26 17:15:13,894 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:15:16,405 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:15:17,758 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:15:21,762 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:15:22,599 DEBUG   SenderThread:3042422 [sender.py:send():382] send: exit
2024-01-26 17:15:22,599 INFO    SenderThread:3042422 [sender.py:send_exit():589] handling exit code: 1
2024-01-26 17:15:22,599 INFO    SenderThread:3042422 [sender.py:send_exit():591] handling runtime: 395
2024-01-26 17:15:22,599 INFO    SenderThread:3042422 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-01-26 17:15:22,599 INFO    SenderThread:3042422 [sender.py:send_exit():597] send defer
2024-01-26 17:15:22,600 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:22,600 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 0
2024-01-26 17:15:22,600 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:22,600 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 0
2024-01-26 17:15:22,600 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 1
2024-01-26 17:15:22,600 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:22,600 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 1
2024-01-26 17:15:22,600 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:22,600 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 1
2024-01-26 17:15:22,600 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 2
2024-01-26 17:15:22,600 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:22,600 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 2
2024-01-26 17:15:22,601 INFO    HandlerThread:3042422 [system_monitor.py:finish():203] Stopping system monitor
2024-01-26 17:15:22,601 INFO    HandlerThread:3042422 [interfaces.py:finish():202] Joined cpu monitor
2024-01-26 17:15:22,601 DEBUG   SystemMonitor:3042422 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-01-26 17:15:22,602 DEBUG   SystemMonitor:3042422 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-01-26 17:15:22,609 INFO    HandlerThread:3042422 [interfaces.py:finish():202] Joined disk monitor
2024-01-26 17:15:23,421 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/output.log
2024-01-26 17:15:23,429 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_created():271] file/dir created: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/wandb-summary.json
2024-01-26 17:15:36,135 INFO    HandlerThread:3042422 [interfaces.py:finish():202] Joined gpu monitor
2024-01-26 17:15:36,136 INFO    HandlerThread:3042422 [interfaces.py:finish():202] Joined memory monitor
2024-01-26 17:15:36,136 INFO    HandlerThread:3042422 [interfaces.py:finish():202] Joined network monitor
2024-01-26 17:15:36,136 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:15:36,137 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:15:36,137 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:15:36,137 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:15:36,137 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:15:36,137 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:36,137 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 2
2024-01-26 17:15:36,137 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 3
2024-01-26 17:15:36,137 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:36,137 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 3
2024-01-26 17:15:36,138 DEBUG   SenderThread:3042422 [sender.py:send():382] send: stats
2024-01-26 17:15:36,139 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:15:36,139 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:36,139 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 3
2024-01-26 17:15:36,139 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 4
2024-01-26 17:15:36,139 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:36,139 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 4
2024-01-26 17:15:36,140 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:36,140 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 4
2024-01-26 17:15:36,140 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 5
2024-01-26 17:15:36,140 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:36,140 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 5
2024-01-26 17:15:36,140 DEBUG   SenderThread:3042422 [sender.py:send():382] send: summary
2024-01-26 17:15:36,140 INFO    SenderThread:3042422 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-01-26 17:15:36,141 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:36,141 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 5
2024-01-26 17:15:36,141 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 6
2024-01-26 17:15:36,141 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:36,141 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 6
2024-01-26 17:15:36,141 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:36,141 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 6
2024-01-26 17:15:36,141 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 7
2024-01-26 17:15:36,141 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:15:36,141 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:36,141 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 7
2024-01-26 17:15:36,142 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:36,142 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 7
2024-01-26 17:15:36,519 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/wandb-summary.json
2024-01-26 17:15:36,631 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:15:37,050 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 8
2024-01-26 17:15:37,050 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:15:37,050 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:37,050 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 8
2024-01-26 17:15:37,050 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:15:37,050 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 8
2024-01-26 17:15:37,051 INFO    SenderThread:3042422 [job_builder.py:build():296] Attempting to build job artifact
2024-01-26 17:15:37,051 INFO    SenderThread:3042422 [job_builder.py:_get_source_type():426] is repo sourced job
2024-01-26 17:15:37,226 INFO    SenderThread:3042422 [job_builder.py:build():402] adding wandb-job metadata file
2024-01-26 17:15:37,229 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 9
2024-01-26 17:15:37,229 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:15:37,229 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 9
2024-01-26 17:15:37,230 DEBUG   SenderThread:3042422 [sender.py:send():382] send: artifact
2024-01-26 17:15:37,519 INFO    Thread-12 :3042422 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/output.log
2024-01-26 17:15:37,632 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:15:38,950 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51d4045d60>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:38,950 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'TF7szjdi1iAfnyx8myEJSw==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '1292'}
2024-01-26 17:15:38,950 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:39,489 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51d4045ee0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:39,490 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'aHPCOiOB0VJCSesalIpFpQ==', 'Content-Type': 'application/json', 'Content-Length': '609'}
2024-01-26 17:15:39,490 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:40,024 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51d4045970>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:40,024 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'TF7szjdi1iAfnyx8myEJSw==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '1292'}
2024-01-26 17:15:40,024 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:40,691 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47d3a30>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:40,691 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'aHPCOiOB0VJCSesalIpFpQ==', 'Content-Type': 'application/json', 'Content-Length': '609'}
2024-01-26 17:15:40,691 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:40,691 INFO    wandb-upload_2:3042422 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f51c47d3a30>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/connectionpool.py", line 799, in urlopen
    retries = retries.increment(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47d3a30>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2758, in upload_file
    response = self._upload_file_session.put(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47d3a30>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2789, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2758, in upload_file
    response = self._upload_file_session.put(
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/yanxindong/anaconda/envs/eva/lib/python3.8/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-01-26 17:15:42,346 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47d3040>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:42,346 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'TF7szjdi1iAfnyx8myEJSw==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '1292'}
2024-01-26 17:15:42,346 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:42,634 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:15:43,008 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c47d3b50>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:43,008 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'aHPCOiOB0VJCSesalIpFpQ==', 'Content-Type': 'application/json', 'Content-Length': '609'}
2024-01-26 17:15:43,008 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:46,728 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51d4045f10>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:46,728 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'TF7szjdi1iAfnyx8myEJSw==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '1292'}
2024-01-26 17:15:46,728 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:47,636 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:15:47,717 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51d4045e20>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:47,717 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'aHPCOiOB0VJCSesalIpFpQ==', 'Content-Type': 'application/json', 'Content-Length': '609'}
2024-01-26 17:15:47,717 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:52,638 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:15:55,871 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c4631a00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:55,871 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'TF7szjdi1iAfnyx8myEJSw==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '1292'}
2024-01-26 17:15:55,871 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:56,099 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c4631c40>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:15:56,099 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'aHPCOiOB0VJCSesalIpFpQ==', 'Content-Type': 'application/json', 'Content-Length': '609'}
2024-01-26 17:15:56,099 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:15:57,640 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:02,642 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:07,644 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:12,228 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c462eaf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:16:12,228 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'TF7szjdi1iAfnyx8myEJSw==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '1292'}
2024-01-26 17:16:12,228 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:16:12,646 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:15,537 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c462a700>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:16:15,537 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'aHPCOiOB0VJCSesalIpFpQ==', 'Content-Type': 'application/json', 'Content-Length': '609'}
2024-01-26 17:16:15,537 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:16:17,649 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:22,651 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:27,653 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:32,656 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:37,658 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:42,660 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:47,662 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:47,730 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c4631fd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:16:47,730 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'aHPCOiOB0VJCSesalIpFpQ==', 'Content-Type': 'application/json', 'Content-Length': '609'}
2024-01-26 17:16:47,730 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:16:50,760 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51d4045490>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:16:50,760 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'TF7szjdi1iAfnyx8myEJSw==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '1292'}
2024-01-26 17:16:50,760 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:16:52,665 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:16:57,667 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:02,670 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:07,672 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:12,674 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:17,675 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:22,677 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:27,680 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:32,682 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:37,691 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:42,693 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:47,695 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:52,697 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:57,700 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:17:59,543 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/6873c23a2381d1524249eb1a948a45a5?Expires=1706346939&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=TP%2FiZ6cRU7bVFRgwIqVk1bu7n1T7%2FhfmxDohD9AKNrDbmJRc0u62Ew7qUSdLzAJj1T1udGpTJdvopiyntZNVJTi0Sn8RZV55aZ3uMn88trPF6YeabEBxtOKzLxKmCmuDcMYSc1qYD0e%2BWA3dCzl6KirtmaSOmMaxDROZg%2FYKYiiD4t8LCH2MCC91h0ShKJM0YqtPBYQaDcn8Rb4fuKn4eG3a%2FJ9SBEcZRHoqZ%2FTjNGavxHI3fTZRKBnN%2FVOZ5OWkjFkKA5ViP9adYnzn%2Fvwxgg%2BbMBqTYd%2FovJfEuqORxBC%2BtJBGYvOGQPHh3IC4Is6EWjgbIBmK2oyueQ3bt4G0kA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c46292e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:17:59,543 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'aHPCOiOB0VJCSesalIpFpQ==', 'Content-Type': 'application/json', 'Content-Length': '609'}
2024-01-26 17:17:59,543 ERROR   wandb-upload_2:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:18:02,702 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:06,432 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/133981200/703789674/4c5eecce3762d6201f9f2c7c9b21094b?Expires=1706346938&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=DYah5qMc2ya17tJlkaKu%2BzdKhJHSh3xtAC5fjXaPGWqo5NP6jOJQe506TjotWFbP%2FDO86noFgrCB9Kda5g7TYuo6BJ8ORu7%2FPTbLne8zWeyzwSc1T6xKQCykyn6DRq5%2BR9SXkVMe1h%2BgA5aBuy49jCi7xPO5A8fk2nl50sMB3gViQGekCHuuHgGZhs9yJD3vP%2FrNDbU%2BpoNL4a5zSXjeGTej06f6tGGABLH0mww0Xa6HP30hJZpmxffenLHe40EI5nL3wbBE34ur4QKwpDYMGnGVmfqPJf7Xv%2BHQDzeqQbkLAHHn1e%2F8vDRdN7ptZBIEQw19BYmrScC249CUmImv7g%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c4634040>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:18:06,433 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'TF7szjdi1iAfnyx8myEJSw==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '1292'}
2024-01-26 17:18:06,433 ERROR   wandb-upload_1:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:18:07,703 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:12,706 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:17,708 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:22,714 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:27,716 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:32,719 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:37,720 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:42,722 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:47,724 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:52,726 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:18:57,728 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:02,730 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:07,732 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:12,734 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:16,326 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2765] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/dynamic_x/bert_pytorch/8pvkre8x/wandb-metadata.json?Expires=1706346538&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IbQwRskqr4JXp2jDiO9931WNVfHZSCjX6g8FBGUndJXRapAXGP1l8iBgjj86AY3CuMJmHkt5YI3V4uL1UjTELnDz%2F%2BdpFt6GufqUQp7fHJJefeCjdHu6jPsRm7qaqPq2T3dlk6SnLH9JUCKutQV4EPqv0bytYkARCZbG5fpgIzSRW%2Fa89RVJgwrPiub1d2nGx9T6bbqHQhXjNUmqABSm1BHDwhawc7xjRXxnNvq%2FcaQeRUxVvZ03Z2nbIMM7FzA77%2FPv7q9rrHMGQzKCD18PNEqGpVGu0e0xsrOajjYP7XqGchpdKKEzWHAiX6M8G9zdhhskwcXPxcqDvanFuO1Sqg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f51c4631d90>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-01-26 17:19:16,326 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2767] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15611'}
2024-01-26 17:19:16,326 ERROR   wandb-upload_0:3042422 [internal_api.py:upload_file():2769] upload_file response body: 
2024-01-26 17:19:17,736 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:22,738 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:27,740 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:32,742 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:37,743 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:42,745 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:47,747 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:52,749 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:19:57,751 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:02,754 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:07,756 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:12,758 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:17,760 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:22,762 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:27,764 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:32,766 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:35,380 INFO    wandb-upload_2:3042422 [upload_job.py:push():89] Uploaded file /home/yanxindong/.local/share/wandb/artifacts/staging/tmp11ls2euf
2024-01-26 17:20:37,768 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:42,771 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:46,194 INFO    wandb-upload_1:3042422 [upload_job.py:push():89] Uploaded file /home/yanxindong/.local/share/wandb/artifacts/staging/tmp7nck7win
2024-01-26 17:20:47,772 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: keepalive
2024-01-26 17:20:48,506 INFO    SenderThread:3042422 [sender.py:send_artifact():1494] sent artifact job-https___github.com_kkkjj-d_mkor.git_bert_run_pretraining.py - {'id': 'QXJ0aWZhY3Q6NzAzNzg5Njc0', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjEzMzk4MTIwMA==', 'latestArtifact': None}}
2024-01-26 17:20:48,507 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:20:48,507 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 9
2024-01-26 17:20:48,507 INFO    SenderThread:3042422 [dir_watcher.py:finish():358] shutting down directory watcher
2024-01-26 17:20:48,507 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:20:48,557 INFO    SenderThread:3042422 [dir_watcher.py:finish():388] scan: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files
2024-01-26 17:20:48,558 INFO    SenderThread:3042422 [dir_watcher.py:finish():402] scan save: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/wandb-summary.json wandb-summary.json
2024-01-26 17:20:48,558 INFO    SenderThread:3042422 [dir_watcher.py:finish():402] scan save: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/conda-environment.yaml conda-environment.yaml
2024-01-26 17:20:48,558 INFO    SenderThread:3042422 [dir_watcher.py:finish():402] scan save: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/wandb-metadata.json wandb-metadata.json
2024-01-26 17:20:48,558 INFO    SenderThread:3042422 [dir_watcher.py:finish():402] scan save: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/output.log output.log
2024-01-26 17:20:48,558 INFO    SenderThread:3042422 [dir_watcher.py:finish():402] scan save: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/requirements.txt requirements.txt
2024-01-26 17:20:48,558 INFO    SenderThread:3042422 [dir_watcher.py:finish():402] scan save: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/config.yaml config.yaml
2024-01-26 17:20:48,558 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 10
2024-01-26 17:20:48,559 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:48,562 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:20:48,562 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 10
2024-01-26 17:20:48,565 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:20:48,565 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 10
2024-01-26 17:20:48,565 INFO    SenderThread:3042422 [file_pusher.py:finish():175] shutting down file pusher
2024-01-26 17:20:48,774 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:48,774 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:49,237 INFO    wandb-upload_1:3042422 [upload_job.py:push():131] Uploaded file /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/conda-environment.yaml
2024-01-26 17:20:49,774 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:49,775 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:49,842 INFO    wandb-upload_5:3042422 [upload_job.py:push():131] Uploaded file /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/config.yaml
2024-01-26 17:20:49,850 INFO    wandb-upload_4:3042422 [upload_job.py:push():131] Uploaded file /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/requirements.txt
2024-01-26 17:20:49,939 INFO    wandb-upload_3:3042422 [upload_job.py:push():131] Uploaded file /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/output.log
2024-01-26 17:20:50,549 INFO    wandb-upload_2:3042422 [upload_job.py:push():131] Uploaded file /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/files/wandb-summary.json
2024-01-26 17:20:50,775 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:50,776 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:51,776 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:51,777 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:52,777 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:52,777 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:53,778 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:20:53,778 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:53,779 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:54,779 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:54,779 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:55,780 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:55,780 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:56,780 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:56,781 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:57,781 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:57,782 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:58,782 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:58,783 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:20:58,783 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:20:59,783 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:20:59,784 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:00,784 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:00,784 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:01,785 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:01,786 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:02,786 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:02,787 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:03,787 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:03,788 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:03,789 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:04,789 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:04,790 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:05,791 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:05,791 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:06,792 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:06,793 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:07,793 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:07,794 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:08,795 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:08,796 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:08,796 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:09,797 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:09,798 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:10,798 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:10,799 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:11,799 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:11,799 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:12,799 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:12,800 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:13,800 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:13,800 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:13,801 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:14,801 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:14,802 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:15,802 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:15,803 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:16,804 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:16,804 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:17,804 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:17,804 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:18,805 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:18,805 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:18,806 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:19,806 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:19,807 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:20,807 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:20,808 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:21,808 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:21,809 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:22,810 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:22,810 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:23,811 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:23,811 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:23,812 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:24,813 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:24,813 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:25,814 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:25,814 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:26,814 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:26,814 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:27,815 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:27,816 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:28,816 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:28,816 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:28,816 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:29,816 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:29,817 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:30,818 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:30,819 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:31,819 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:31,820 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:32,821 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:32,822 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:33,822 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:33,823 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:33,823 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:34,823 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:34,824 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:35,824 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:35,824 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:36,825 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:36,826 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:37,826 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:37,827 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:38,827 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:38,827 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:38,828 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:39,827 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:39,828 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:40,830 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:40,830 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:41,830 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:41,830 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:42,831 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:42,831 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:43,831 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:43,832 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:43,832 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:44,832 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:44,832 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:45,832 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:45,832 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:46,833 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:46,834 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:47,833 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:47,834 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:48,834 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:48,835 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:48,835 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:49,835 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:49,836 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:50,836 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:50,836 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:51,837 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:51,837 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:52,838 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:52,838 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:53,839 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:53,839 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:53,840 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:54,841 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:54,842 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:55,842 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:55,843 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:56,843 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:56,844 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:57,844 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:57,845 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:58,845 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:58,846 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:21:58,846 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:21:59,846 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:21:59,847 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:00,847 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:00,848 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:01,848 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:01,849 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:02,849 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:02,849 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:03,849 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:03,850 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:03,851 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:04,849 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:04,851 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:05,850 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:05,851 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:06,852 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:06,853 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:07,852 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:07,852 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:08,852 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:08,854 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:08,855 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:09,853 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:09,854 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:10,854 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:10,854 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:11,854 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:11,854 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:12,855 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:12,856 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:13,856 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:13,857 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:13,857 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:14,858 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:14,859 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:15,859 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:15,860 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:16,861 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:16,861 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:17,861 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:17,862 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:18,863 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:18,863 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:18,864 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:19,863 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:19,864 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:20,864 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:20,865 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:21,864 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:21,865 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:22,866 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:22,867 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:23,866 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:23,867 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:23,867 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:24,867 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:24,868 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:25,868 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:25,868 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:26,868 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:26,869 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:27,868 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:27,870 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:28,870 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:28,870 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:28,871 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:29,870 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:29,871 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:30,871 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:30,872 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:31,872 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:31,872 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:32,873 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:32,874 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:33,874 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:33,874 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:33,875 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:34,875 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:34,875 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:35,876 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:35,877 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:36,877 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:36,878 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:37,878 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:37,879 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:38,878 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:38,879 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:38,879 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:39,880 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:39,880 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:40,881 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:40,882 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:41,881 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:41,882 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:42,882 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:42,883 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:43,883 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:43,884 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:43,885 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:44,884 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:44,884 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:45,884 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:45,885 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:46,886 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:46,886 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:47,887 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:47,888 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:48,888 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:48,889 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:48,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:49,889 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:49,890 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:50,890 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:50,891 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:51,891 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:51,892 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:52,891 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:52,892 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:53,893 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:53,893 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:53,894 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:54,893 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:54,894 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:55,894 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:55,894 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:56,894 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:56,895 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:57,894 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:57,895 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:58,896 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:22:58,896 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:58,897 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:22:59,897 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:22:59,898 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:00,898 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:00,899 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:01,899 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:01,899 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:02,900 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:02,900 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:03,900 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:03,901 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:03,902 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:04,902 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:04,903 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:05,903 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:05,904 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:06,904 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:06,905 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:07,905 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:07,906 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:08,906 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:08,907 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:08,907 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:09,907 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:09,908 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:10,908 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:10,909 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:11,909 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:11,910 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:12,910 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:12,910 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:13,910 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:13,910 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:13,910 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:14,911 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:14,911 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:15,911 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:15,912 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:16,913 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:16,913 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:17,913 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:17,914 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:18,914 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:18,915 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:18,915 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:19,915 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:19,916 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:20,916 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:20,916 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:21,916 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:21,917 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:22,917 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:22,918 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:23,918 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:23,919 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:23,919 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:24,919 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:24,919 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:25,919 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:25,919 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:26,920 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:26,920 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:27,921 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:27,921 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:28,921 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:28,922 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:28,923 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:29,922 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:29,923 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:30,923 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:30,924 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:31,924 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:31,925 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:32,925 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:32,926 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:33,926 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:33,926 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:33,927 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:34,926 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:34,927 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:35,926 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:35,927 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:36,927 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:36,928 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:37,928 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:37,929 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:38,929 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:38,930 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:38,930 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:39,930 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:39,930 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:40,930 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:40,931 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:41,931 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:41,932 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:42,931 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:42,932 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:43,933 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:43,933 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:43,934 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:44,933 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:44,933 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:45,934 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:45,934 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:46,935 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:46,935 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:47,935 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:47,936 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:48,936 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:48,937 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:48,937 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:49,937 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:49,938 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:50,937 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:50,938 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:51,938 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:51,938 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:52,938 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:52,939 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:53,940 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:53,940 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:53,941 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:54,940 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:54,941 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:55,941 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:55,941 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:56,942 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:56,943 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:57,943 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:57,943 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:58,943 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:58,944 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:23:58,944 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:23:59,944 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:23:59,945 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:00,946 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:00,946 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:01,946 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:01,947 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:02,948 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:02,948 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:03,949 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:03,950 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:24:03,950 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:04,950 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:04,951 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:05,951 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:05,951 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:06,952 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:06,952 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:07,952 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:07,953 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:08,954 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:08,954 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:24:08,955 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:09,955 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:09,955 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:10,956 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:10,956 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:11,957 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:11,958 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:12,958 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:12,959 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:13,959 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:13,960 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:24:13,960 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:14,960 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:14,961 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:15,961 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:15,962 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:16,963 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:16,964 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:17,964 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:17,964 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:18,964 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:18,964 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:18,965 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:24:19,964 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:19,965 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:20,966 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:20,966 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:21,966 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:21,966 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:22,967 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:22,967 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:23,967 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:23,968 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: status_report
2024-01-26 17:24:23,968 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:24,969 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:24,970 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:25,970 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:25,971 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:26,162 INFO    wandb-upload_0:3042422 [upload_job.py:push():131] Uploaded file /tmp/tmp77s5a5vtwandb/jymp1u44-wandb-metadata.json
2024-01-26 17:24:26,362 INFO    Thread-11 :3042422 [sender.py:transition_state():617] send defer: 11
2024-01-26 17:24:26,363 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:24:26,363 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 11
2024-01-26 17:24:26,363 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:24:26,363 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 11
2024-01-26 17:24:26,363 INFO    SenderThread:3042422 [file_pusher.py:join():181] waiting for file pusher
2024-01-26 17:24:26,363 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 12
2024-01-26 17:24:26,363 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:24:26,363 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 12
2024-01-26 17:24:26,363 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:24:26,363 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 12
2024-01-26 17:24:26,363 INFO    SenderThread:3042422 [file_stream.py:finish():595] file stream finish called
2024-01-26 17:24:26,638 INFO    SenderThread:3042422 [file_stream.py:finish():599] file stream finish is done
2024-01-26 17:24:26,639 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 13
2024-01-26 17:24:26,639 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:24:26,639 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 13
2024-01-26 17:24:26,640 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:24:26,640 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 13
2024-01-26 17:24:26,640 INFO    SenderThread:3042422 [sender.py:transition_state():617] send defer: 14
2024-01-26 17:24:26,640 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: defer
2024-01-26 17:24:26,641 INFO    HandlerThread:3042422 [handler.py:handle_request_defer():172] handle defer: 14
2024-01-26 17:24:26,641 DEBUG   SenderThread:3042422 [sender.py:send():382] send: final
2024-01-26 17:24:26,641 DEBUG   SenderThread:3042422 [sender.py:send():382] send: footer
2024-01-26 17:24:26,642 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: defer
2024-01-26 17:24:26,642 INFO    SenderThread:3042422 [sender.py:send_request_defer():613] handle sender defer: 14
2024-01-26 17:24:26,644 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:26,644 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: poll_exit
2024-01-26 17:24:26,644 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: server_info
2024-01-26 17:24:26,644 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: get_summary
2024-01-26 17:24:26,644 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: sampled_history
2024-01-26 17:24:26,645 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-26 17:24:26,645 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:26,646 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: poll_exit
2024-01-26 17:24:26,646 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: server_info
2024-01-26 17:24:26,649 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: job_info
2024-01-26 17:24:26,911 DEBUG   SenderThread:3042422 [sender.py:send_request():409] send_request: job_info
2024-01-26 17:24:26,912 INFO    MainThread:3042422 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-01-26 17:24:26,912 INFO    MainThread:3042422 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-01-26 17:24:26,912 INFO    MainThread:3042422 [wandb_run.py:_footer_sync_info():3816] logging synced files
2024-01-26 17:24:26,913 DEBUG   HandlerThread:3042422 [handler.py:handle_request():146] handle_request: shutdown
2024-01-26 17:24:26,913 INFO    HandlerThread:3042422 [handler.py:finish():869] shutting down handler
2024-01-26 17:24:27,650 INFO    WriterThread:3042422 [datastore.py:close():296] close: /home/yanxindong/mkor/bert/wandb/run-20240126_170845-8pvkre8x/run-8pvkre8x.wandb
2024-01-26 17:24:27,911 INFO    SenderThread:3042422 [sender.py:finish():1572] shutting down sender
2024-01-26 17:24:27,912 INFO    SenderThread:3042422 [file_pusher.py:finish():175] shutting down file pusher
2024-01-26 17:24:27,912 INFO    SenderThread:3042422 [file_pusher.py:join():181] waiting for file pusher
